mkosi/run.py

```diff
@@ -306,10 +306,12 @@ def bwrap_cmd(
         "--dev-bind", "/", "/",
         "--chdir", Path.cwd(),
         "--die-with-parent",
+        "--ro-bind", (root or Path("/")) / "usr", "/usr",
     ]
 
-    if root:
-        cmdline += ["--bind", root / "usr", "/usr"]
+    for p in ("/etc", "/opt", "/srv", "/root", "/boot", "/efi"):
```

Comment:
conceivably on a build vm/chroot someone might be executing from under /root, so maybe first check that the working dir is not under it?

Reply:
On a similar note: With a check for the current working directory, this could also include `/home`? 

Protecting at least the unnecessary (for us) bits of `/var` might also be a good idea.